core/azure-core/src/main/java/com/azure/core/polling/PollerOptions.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.polling;
+
+import java.time.Duration;
+
+/**
+ * This class provide configuration options needed to create Poller.
+ *
+ * <p><strong>Place holder for Poller Options</strong></p>
+ */
+public class PollerOptions {
+
+    private String negativeValueFormat = "Null, negative or zero poll interval not allowed.";
+    private Duration pollInterval;
+
+    /**
+     * constructor
+     * @param pollInterval It ensure that polling happens only once in given pollInterval.
+     *
+     * @throws IllegalArgumentException for {@code null} , zero and negative values.
+     */
+    public PollerOptions(Duration pollInterval) throws IllegalArgumentException {
+        validateValuesAndThrow(pollInterval);
```

Comment:
Same comment here - get rid of the method and use `Objects` API / extra conditional code in the constructor directly instead.

Reply:
will do